src/server/array_func.c

```diff
@@ -791,29 +757,28 @@ setup_arrayjob_attrs(attribute *pattr, void *pobj, int mode)
 int
 fixup_arrayindicies(attribute *pattr, void *pobj, int mode)
 {
-	int   i;
-	int   x, y, z, ct;
+	int i;
+	int x, y, z, ct;
```

Comment:
Break the declaration to multi-line. And wouldn't it be better to use something meaningfull than x,y,z?